x-pack/solutions/observability/plugins/slo/test/scout/ui/tests/slos_overview.spec.ts

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { expect } from '@kbn/scout-oblt';
+import { test } from '../fixtures';
+
+const TEST_TIMEOUT = 3 * 60 * 1000; // 3 minutes timeout, needed to wait for the SLOs to be created
+
+test.describe('SLOs Overview', { tag: ['@ess', '@svlOblt'] }, () => {
+  test.describe.configure({ timeout: TEST_TIMEOUT });
+
+  test.beforeAll(async ({ sloData }) => {
+    await sloData.generateSloData();
+    await sloData.addSLO();
+  });
+
+  test.beforeEach(async ({ pageObjects, browserAuth }) => {
+    await browserAuth.loginAsAdmin();
+
+    await pageObjects.slo.goto();
+  });
+
+  test('Go to slos overview', async ({ page }) => {
+    // Already navigated in beforeEach
+    // This test ensures the page loads
+    expect(page).toBeDefined();
+  });
+
+  test('validate data retention tab', async ({ page }) => {
+    await expect(async () => {
+      await page.getByTestId('querySubmitButton').click();
+
+      await expect
+        .poll(() => page.locator('text=Test Stack SLO').count(), { timeout: 1000 })
+        .toBeGreaterThan(5);
+    }).toPass({
+      intervals: [10000],
+      timeout: TEST_TIMEOUT,
+    });
+  });
```

Comment:
Maybe it makes sense to combine it into a single `test` block? It does look like a single flow and 2nd scenario won't work if page is not loaded anyways

Reply:
Done